src/node_worker.cc

```diff
@@ -223,6 +238,7 @@ size_t Worker::NearHeapLimit(void* data, size_t current_heap_limit,
                              size_t initial_heap_limit) {
   Worker* worker = static_cast<Worker*>(data);
   worker->custom_error_ = "ERR_WORKER_OUT_OF_MEMORY";
+  worker->custom_error_str_ = "JS heap Out of Memory";
```

Comment:
```suggestion
  worker->custom_error_str_ = "JS Heap Out of Memory";
```

or

```suggestion
  worker->custom_error_str_ = "JS heap out of memory";
```

But I’d prefer to be consistent :)

Reply:
@addaleax, thanks. Made it consistent.